distribution/build.gradle

```diff
@@ -388,9 +390,7 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
         // module provided bin files
         with copySpec {
           eachFile { it.setMode(0755) }
-          if(testDistro == false) {
-            from(defaultBinFiles)
-          }
+          from(defaultBinFiles)
```

Comment:
For test distros we don't need _all_ the bin files right? Just the ones from the security module.

Reply:
Yes, you are right. To be more precise we just need `elasticsearch-cli`, `x-pack-env` and` x-pack-security-env`, maybe I can qualify this to just that for `testDistro`. I will amend